modules/nextflow/src/main/groovy/nextflow/processor/TaskConfig.groovy

```diff
@@ -429,24 +436,6 @@ class TaskConfig extends LazyMap implements Cloneable {
         return null
     }
 
-    /**
-     * @return Parse the {@code clusterOptions} configuration option and return the entries as a list of values
-     */
-    List<String> getClusterOptionsAsList() {
-
-        def opts = get('clusterOptions')
-        if ( !opts ) {
-            return Collections.emptyList()
-        }
-
-        if( opts instanceof Collection ) {
-            return new ArrayList<String>(opts)
-        }
-        else {
-            return CmdLineHelper.splitter( opts.toString() )
-        }
-    }
-
```

Comment:
It turns out list was already supported

Reply:
yes, but not in the way we wanted. you had to specify each option/value separately and you'd have to include a blank string for options with no value, which nobody knew about and is what led to this issue. considering it wasn't documented, I think it's fine to change.

I ended up removing this method and adding `getClusterOptionsAsString()` because most executors just add the cluster options as a single line, whereas only a few executors use this list form and they all do it differently. This method was used only by LSF for instance